### PR TITLE
Implement "Product Feed" feed type of SubmitFeed

### PR DESCRIPTION
**Pull request by sachinmaharjan**
_POST_PRODUCT_DATA_
Story: https://jira.blurb.com/browse/WT-7093

This will enable us to upload book data amazon mws.

This does not upload following fields
- Image
- Pages
- Will ship internationally
- Quantity
- Price

Remaining work for feed:
```
_POST_PRODUCT_PRICING_DATA_
_POST_PRODUCT_IMAGE_DATA_
_POST_INVENTORY_AVAILABILITY_DATA_
```

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1 +1,6 @@
 require "bundler/gem_tasks"
+
+desc "Open an irb session preloaded with this library"
+task :console do
+  sh "irb -rubygems -I lib -r ruby-mws.rb"
```
Comment (pierrel) [inline Rakefile]: Nice! Can you add this to the README?

**Comment by pierrel** [comment]
Looks good. Just one comment. Merge in when you get the chance.

File: Rakefile (hunk 1 above)
Comment (sachinmaharjan) [inline Rakefile]: Added

**Comment by nhullrichter-blurb** [comment]
sgtm, though I'm not really that familiar with this codebase.

**Pull request merged**